tests/test_bin.py

```diff
@@ -132,6 +132,16 @@ def test_bin_hy_stdin_except_do():
     assert "zzz" in output
 
 
+def test_bin_hy_stdin_bad_repr():
+    # https://github.com/hylang/hy/issues/1389
+    output, err = run_cmd("hy", "\n".join([
+        "(defclass BadRepr [] (defn __repr__ [self] (/ 0)))",
+        "(BadRepr)",
+        '(+ "A" "Z")']))
```

Comment:
Python does automatically concatenate adjacent string literals, you know, there's no need to join them. And triple-quoted strings can contain newlines without any escapes.

Reply:
I was working around Flake8, but I forgot about triple quotes.